### PR TITLE
Project documentation TempTune

### PR DESCRIPTION
**Pull request by Gabriel-x-1**
### Pull Request Overview

This pull request adds the documentation of the project 

### Author

Signed-off-by: Bilciurescu Gabriel <gabi2015a1b1@gmail.com>
Signed-off-by: Pica Eduard <eduard_ionut.pica@stud.fils.upb.ro>

### CODE REVIEW

**Review by alexandruradovici — COMMENTED** [review]
Isn't this a duplicate?

**Comment by Gabriel-x-1** [comment]
TempTune is a team project, i am collaborating with Pica Eduard on this. Should we make a single pull request?

File: docs/project/2024/gabriel.bilciurescu/index.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+| [Breadboard](https://www.optimusdigital.ro/ro/prototipare-breadboard-uri/8-breadboard-830-points.html?search_query=breadboard&results=143) | for prototyping | [10 RON](https://www.optimusdigital.ro/ro/prototipare-breadboard-uri/8-breadboard-830-points.html?search_query=breadboard&results=143) |
+| [2n2222 NPN BJT](https://pdf1.alldatasheet.com/datasheet-pdf/view/15067/PHILIPS/2N2222.html) | NPN BJT Transistors for current amplification | [2 RON](https://www.optimusdigital.ro/ro/componente-electronice-tranzistoare/935-tranzistor-s9013-npn-50-pcs-set.html?search_query=2n2222&results=9) |
+| [TP4056](https://pdf1.alldatasheet.com/datasheet-pdf/view/1487471/ETC2/TP4056.html) | Charging module for the battery | [6 RON](https://www.optimusdigital.ro/ro/electronica-de-putere-incarcatoare/7534-incarcator-tp4056-cu-micro-usb-pt-baterie-lipo-1a-cu-protectie-pentru-circuite.html?search_query=modul+incarcator+&results=37) |
+| [18650 LI-ION Battery 3200mA 10A Panasonic]() |  Battery | [23 RON](https://www.emag.ro/acumulator-18650-li-ion-3200ma-10a-panasonic-lincr18650bd/pd/DL86WWBBM/) |
```
Comment (alexandruradovici) [inline docs/project/2024/gabriel.bilciurescu/index.md:83]: The link is missing here.

File: docs/project/2024/gabriel.bilciurescu/arhitecture.png
Comment (alexandruradovici) [inline docs/project/2024/gabriel.bilciurescu/arhitecture.png:1]: Rename the folder with both of your usernames with _ between them.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
This is the only PR that you sent for this project, right?

**Comment by alexandruradovici** [comment]
Add another Signed-off line to the pull request description with the second name and email.

**Comment by alexandruradovici** [comment]
> TempTune is a team project, i am collaborating with Pica Eduard on this. Should we make a single pull request?

Yes

**Comment by Gabriel-x-1** [comment]
solved the requested changes

**Review by alexandruradovici — APPROVED** [review]

**Review by DTeodor-Alexaandru — APPROVED** [review]

**Pull request merged**